dremio_client/dremio_simple_client.py

```diff
@@ -423,9 +423,21 @@ def graph(self, cid):
         return graph(self._token, self._base_url, cid, ssl_verify=self._ssl_verify)
 
     def refresh_vds_reflection_by_path(self, path):
-        """ Refresh the metadata for a given physical dataset
+        """ Refresh the reflection for a given virtual dataset
 
         :param path: list ['space', 'folder', 'vds']
         :return: None
         """
         return refresh_vds_reflection_by_path(self, path)
+
+    def refresh_reflections_of_one_dataset(self, path):
```

Comment:
This is a potentially destructive operation as it temporarily removes a reflection causing a fair chunk of work do be done and slowing down queries in the meantime. I think it should have a [big warning](https://pythonhosted.org/an_example_pypi_project/sphinx.html#paragraph-markup) in the pydoc and perhaps a `logging.warn` when the method is called.